src/components/Calendar/RangePicker.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import styled from 'react-emotion';
+import { DateRangePicker } from 'react-dates';
+import 'react-dates/initialize';
+
+import { CalendarWrap } from './components';
```

Comment:
Annoying comment from me: could you make `CalendarWrapper`? I think throughout the codebase we tend to call these things wrappers as opposed to wrap.

Reply:
Sure. I am the worst with naming. Appreciated :)